web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -43,6 +46,9 @@ public class ValidationController {
   @Autowired private StorageHelper storageHelper;
   @Autowired private ValidationHandler validationHandler;
 
+  @Autowired private VersionResolver checker;
```

Comment:
Rename it to increase readability
```suggestion
  @Autowired private VersionResolver versionResolver;
```